.github/workflows/build.yml

```diff
@@ -439,3 +439,39 @@ jobs:
 
           echo "Failed to install expected version $VERSION"
           exit 1
+
+  # finally, containerize the package and upload to the GHCR
+  build-and-push-to-ghcr:
+    needs: package
+    if: github.event_name == 'release' && github.event.release.prerelease == 'false'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+      - name: Extract Release Version
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+        shell: bash
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push Docker image
+        id: docker_build
+        uses: docker/build-push-action@v3
+        with:
+          push: true
+          tags: ghcr.io/sage-bionetworks/synapsepythonclient:latest,ghcr.io/sage-bionetworks/synapsepythonclient:${{ env.RELEASE_VERSION }}
```

Comment:
What currently uses the synapseclient docker container (the older non-ghcr versions)? I'm guessing it won't be needed to be tagged for `develop` or feature branches?

Reply:
> What currently uses the synapseclient docker container (the older non-ghcr versions)?

I'm not sure myself, but we are only going to be publishing on GHCR from now on as previous solutions were manual.

> I'm guessing it won't be needed to be tagged for develop or feature branches?

This job is only run during the package step and when `[github.event.release.prerelease == 'false'`](if: github.event_name == 'release' && github.event.release.prerelease == 'false')`. These 2 things make sure that we're only releasing tags when we create them. Based on what I am seeing in the code Jenny wrote is that it'll automatically pull the version of the tag and use that. Which is what we want.

Reply:
Gotcha, yeah that makes sense if this is only needed during package release step